examples/nitclk/multi-device-configured-acquisition-tclk.py

```diff
@@ -0,0 +1,294 @@
+#
+# This is an example of plotting waveforms read from an NI-SCOPE device through gRPC
+# Tested with a 100 kHz tone input to channel 0
```

Comment:
I think this line of Tested with 100kHz input is also from scope example. You can discuss with @harsha-bhushan if same applies to this example or not